bin/smart-proxy-dhcp-isc

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
```

Comment:
why was this file added and how does it fix the `Style/RedundantConstantBase` cop?

not that I mind adding it, just puzzled how it fits into this PR